src/server/pps/server/s3g_sidecar_test.go

```diff
@@ -666,14 +664,14 @@ func TestS3SkippedDatums(t *testing.T) {
 		pipelineCommit := client.NewCommit(pipeline, "master", "")
 		// Add files to 'repo'. Old files in 'repo' should be reprocessed in every
 		// job, changing the 'background' field in the output
-		for i := 0; i < 10; i++ {
+		for i := 1; i < 7; i++ {
```

Comment:
I also set `i` to start at 1 to make the counting a little more readable now that we start with 1 ALIAS commit